src/services/control.js

```diff
@@ -305,49 +327,62 @@ var gitPull = function (repoPath, callback) {
 // Read functions
 
 var readContentMap = function (callback) {
-  var contentMapPath = PathService.getConfigPath(config.control_content_file());
+  var contentFiles = globby.sync([
+    PathService.getConfigPath(config.control_content_file()),
+    PathService.getConfigPath('content.d/**/*')
+  ]);
```

Comment:
> Are those environment variables being explicitly set when we run the presenter in production?

Oh good thought. [It looks like no](https://github.com/deconst/deploy/blob/master/roles/worker/files/services/deconst-presenter%40.service#L16-L25) so we should be good :sparkles: